### PR TITLE
Fixing config.yml param "bin_path" - bin path is now undepending on t…

### PR DESCRIPTION
**Pull request by linslin**
An other windows fix, could you please check if this still works on unix? - https://github.com/linslin/Yii2-Curl/issues/57

### CODE REVIEW

**Comment by mcustiel** [comment]
Hey @linslin, thank you for your help with this project.
There is a merge conflict, could you merge my master to your fork?

**Comment by linslin** [comment]
Oh yea, you are right. Let me try again.

File: src/Extension/PhiremockProcess.php [hunk 1]
```diff
@@ -47,6 +47,7 @@ class PhiremockProcess
      */
     public function start($ip, $port, $path, $logsPath, $debug)
     {
+        $path = realpath($path);
```
Comment (mcustiel) [inline src/Extension/PhiremockProcess.php]: What do you think about adding the call to realpath when the start method is being called in Phiremock.php file? That way, this method is agnostic about how the path is built.
Would also be great to call it for logsPath variable.

**Comment by linslin** [comment]
Allright, conflict should be solved.

File: src/Extension/PhiremockProcess.php (hunk 1 above)
Comment (linslin) [inline src/Extension/PhiremockProcess.php]: Absolutly, we thought about the logFilePathe too but we needed a quick fix for the bin_path first. You are more familiar with your codes, so please decide it by yourself. =)

Comment (mcustiel) [inline src/Extension/PhiremockProcess.php]: Ok. Will merge it and later move it outside the method also for logspath.

Comment (linslin) [inline src/Extension/PhiremockProcess.php]: Great! Thx m8. Followed you up =)
